tools/magfit_WMM.py

```diff
@@ -181,6 +181,9 @@ def wmm_error(p):
 
     return ret
 
+def printProgress(x):
+    print("offsets:", x[0:3], " scale:", x[3])
```

Comment:
This isn't the way one usually prints stuff in Python now-adays :-)
```suggestion
    print(f"offsets: {x[0:3]} scale: {x[3]}")
```
would be the modern way